src/resources/elements/typefield.js

```diff
@@ -92,18 +92,18 @@ export class Typefield extends Field {
   init(id = '', args = {}) {
     args = Object.assign({
```

Comment:
I've added JSDoc-formatted comments to `init()` explaining all the `Typefield`-specific contents of `args`